### PR TITLE
Import South Buckinghamshire (closes #2349)

### LINKED ISSUE #2349: Import E07000006-South Buckinghamshire District Council
Xpress DC

### PR DESCRIPTION
**Pull request by alexsdutton**

### CODE REVIEW

File: polling_stations/apps/data_collection/management/commands/import_south_bucks.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    council_id = "E07000006"
+    addresses_name = (
+        "parl.2019-12-12/Version 1/Democracy_Club__12December2019 Beaconsfield.tsv"
+    )
+    stations_name = (
+        "parl.2019-12-12/Version 1/Democracy_Club__12December2019 Beaconsfield.tsv"
+    )
+    elections = ["parl.2019-12-12"]
+    csv_delimiter = "\t"
+    # allow_station_point_from_postcode = False
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_south_bucks.py]:
```suggestion
    allow_station_point_from_postcode = False
```

File: polling_stations/apps/data_collection/management/commands/import_south_bucks.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            return None  # looks odd
+
+        rec = super().address_record_to_dict(record)
+        uprn = record.property_urn.strip().lstrip("0")
+
+        if uprn in [
+            "100081076380",  # SL23HW -> SL23HJ : Brook House, Templewood Lane, Stoke Poges, Slough
+            "10003236726",  # UB94DQ -> UB94DG : Flat 4 The Old Mill, Oxford Road, Denham, Uxbridge
+            "10003236727",  # UB94DQ -> UB94DG : Flat 6 The Old Mill, Oxford Road, Denham, Uxbridge
+            "10003236729",  # UB94DQ -> UB94DG : Flat 8 The Old Mill, Oxford Road, Denham, Uxbridge
```
Comment (chris48s) [inline polling_stations/apps/data_collection/management/commands/import_south_bucks.py]: You sure about these 3? Have a look at them on `dashboard/postcode/UB94DG/` and `dashboard/postcode/UB94DQ/`

File: polling_stations/apps/data_collection/management/commands/import_south_bucks.py (hunk 1 above)
Comment (alexsdutton) [reply polling_stations/apps/data_collection/management/commands/import_south_bucks.py]: :+1:

File: polling_stations/apps/data_collection/management/commands/import_south_bucks.py (hunk 2 above)
Comment (alexsdutton) [reply polling_stations/apps/data_collection/management/commands/import_south_bucks.py]: Not now you mention it ;-). Now noped.

**Pull request merged**